__docs__/wonder-blocks-dropdown/multi-select.stories.tsx

```diff
@@ -650,3 +655,51 @@ export const CustomOptionItems: StoryComponentType = {
         ),
     ],
 };
+
+/**
+ * This example illustrates how a JSX Element can appear as the label when only
+ * one option is selected and `labelAsText` is undefined.
+ * **Note** this is only supported for SingleSelect and MultiSelect, not Combobox.
+ */
+export const CustomOptionItemsWithoutTextLabel: StoryComponentType = {
```

Comment:
Maybe it's just me, but I found the "without text label" part of these new stories a little confusing! 😅  

Would something like "with node label" or "with child element as label" reflect the unique aspect of these updates?

Reply:
Yes, good suggestion! just pushed an update